kie-spring-boot/kie-spring-boot-samples/kie-server-spring-boot-sample/src/test/java/org/kie/server/springboot/samples/KafkaFixture.java

```diff
@@ -230,7 +228,7 @@ protected Properties producerProps(String bootstrapServer, boolean transactional
     }
     
     protected ListAppender<ILoggingEvent> addLogAppender() {
-        Logger logger = (Logger) LoggerFactory.getLogger(KafkaServerExtension.class);
+        Logger logger = (Logger) LoggerFactory.getLogger(KafkaServerExtension.class.getPackage().getName());
```

Comment:
this change is not needed and goes against all the other loggers. remove it.

Reply:
I have to change that because we are using per logger class and test was expecting logs in KieServer

Reply:
@elguardian In this case, the purpose of the test is to check that the exception is logged, no matter which the logger is associated with, so I think the change is valid after refactoring the extension

Reply:
fine. it does belong to tests not really prod code